src/components/AddDataInstructions.tsx

```diff
@@ -11,8 +14,69 @@ export const AddDataInstructions = ({
 }: AddDataInstructionsProps) => (
   <Panel>
     <PanelBody>
-      <p>TODO: add data instructions</p>
-      {children}
+      <article className="add-data-instructions">
+        <h3>Add data to the PDC</h3>
+
+        <p>
+          <b>Adding data via the bulk uploader</b>
+          {' '}
+          is drag-and-drop easy:
+        </p>
+
+        <ol>
+          <li>
+            Create a CSV of your data with base field keys as the first row
```

Comment:
After talking with @slifty about implementing the CSV parser, it sounds like there are some additional requirements here: the first two fields _must_ be `proposal_external_id` and `applicant_external_id`, and those are not base fields. That unfortunately complicates the instructions. I'm okay with deferring those instruction changes for now, but we'll need to follow up on that.

Reply:
+1 on the idea of not changing this PR based on that just yet; I don't know exactly what it's going to look like yet so we should wait until the PR is at least opened.

Reply:
Oof… I trust you guys, but to pretend I don't know how computers work and am just a "make UX nice" guy:

1. The positional requirement, while easiest for a user to grok/resolve, is weird to me. There are CSV parsers that create objects for each row, keyed by column name, rather than positional arrays; are we avoiding that for memory consumption reasons or a tooling preference or…?
2. I think saying "must be base fields PLUS these two fields" is fine, but are the values of those fields something they get to make up, or do they have to reference correct values from our database (like a foreign key reference or something)? If the former, fine. We can adapt the instructions I'm sure. But if the latter, that's a real wrinkle in the UX and cuts down the audience for this tool dramatically.

Reply:
> The positional requirement, while easiest for a user to grok/resolve, is weird to me.

As @slifty said, we're not exactly sure what it's going to look like yet, so what I wrote is what I imagined from our conversation. We could definitely make it so those special fields are allowed anywhere; I thought requiring them at the beginning would be easier to understand and easier to explain, but I don't think there are any technical requirements there, and I'm happy to defer to your UX expertise on that!

> are the values of those fields something they get to make up?

Yes! The applicants will be upserted (created or reused) based on the external ID; the proposal external ID must be unique within the CSV.

(In more detail: there is a `UNIQUE` constraint on `proposals (applicant_id, opportunity_id, external_id)`. I believe we'll be creating a new `opportunity_id` for each CSV, and applicants are upserted, so all that's left to satisfy that constraint is making sure there aren't two proposals with the same `external_id`.)